java/src/main/java/ai/rapids/cudf/CompressionMetadataWriterOptions.java

```diff
@@ -24,11 +24,13 @@
 public class CompressionMetadataWriterOptions extends ColumnWriterOptions.StructColumnWriterOptions {
   private final CompressionType compressionType;
   private final Map<String, String> metadata;
+  private  final int stripeSizeRows;
```

Comment:
It is not a godd idea to add this "stripeSizeRows" item into this common class "CompressionMetadataWriterOptions" if "stripeSizeRows" is specific to ORC. 
Better to add it into a more specific class, e.g. ORCWriteOption